spec/project/project_requires_spec.rb

```diff
@@ -1,8 +1,19 @@
 RSpec.describe 'Project requires' do
+  # rubocop:disable RSpec/MultipleExpectations
   it 'alphabetizes cop requires' do
-    source   = SpecHelper::ROOT.join('lib', 'rubocop-rspec.rb').read
-    requires = source.split("\n").grep(%r{rubocop/cop/rspec/[^(?:cop)]})
+    source   = SpecHelper::ROOT.join('lib', 'rubocop', 'cop', 'rspec_cops.rb')
+    captures = source.read.scan(%r{^(require_relative 'rspec/(.*?/)?(.*?)')$})
 
-    expect(requires.join("\n")).to eql(requires.sort.join("\n"))
+    # Sanity check in case these move again.
```

Comment:
I think this comment will provide less and less information with time. Perhaps remove “again” and describe “these” in words?

Reply:
Sure.